### DISCUSSION TITLE
Kubernetes update

### DISCUSSION THREAD

**Discussion by Lory1990** [mia-platform/community#339] [General]
### Description

Hi erveyone, 
is kubernetes version 1.27.3  supported?

<img width="848" alt="image" src="https://github.com/mia-platform/community/assets/7715601/c0b771f0-555d-47db-864d-52bc342bc88d">

**Comment by davidebianchi** [comment]
Hi @Lory1990! Yes, v1.27.x are supported versions. Here the compatibility matrix: https://docs.mia-platform.eu/docs/preview/self_hosted/self_hosted_requirements#software-and-hardware-prerequisites

**Reply by Lory1990** [reply]
Fantastic thank you!
